next.config.js

```diff
@@ -14,6 +14,10 @@ const securityHeaders = [
 
 module.exports = withKeystone(
   withBundleAnalyzer({
+    publicRuntimeConfig: {
```

Comment:
@esacteksab this lets us define env vars for NextJS pages that can be defined at runtime (instead of build)